spec/tile-spec.yml

```diff
@@ -0,0 +1,245 @@
+  - name: Healthcheck
+    description: |
+      Assurance that the tile server is still alive
+
+paths:
+  /healthcheck/:
+    x-resource: Healthcheck
+    get:
+      summary: 'Confirm the tile server is available'
+      tags:
+        - Healthcheck
+      responses:
+        200:
+          description: 'OK'
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+              reason:
+                type: array
+                items:
+                  type: string
+
+  /tools/{analysisId}/{z}/{x}/{y}/:
+    x-resource: Anslysis
+    get:
+      produces:
+        - image/png
+      summary: Visualize an analysis (or node of an analysis) as a tile
+      tags:
+        - Analysis
+      parameters:
+        - $ref: '#/parameters/zoom'
+        - $ref: '#/parameters/x'
+        - $ref: '#/parameters/y'
+        - $ref: '#/parameters/token'
+        - $ref: '#/parameters/mapToken'
+        - name: analysisId
+          description: 'ID of the analysis to visualize'
+          in: path
+          type: string
+          format: uuid
+          required: true
+        - name: node
+          description: 'ID of the subnode of this analysis to visualize'
+          in: query
+          type: string
+          format: uuid
+          required: false
+      responses:
+        200:
+          description: OK
+        401:
+          description: 'Authentication failed -- no authentication methods succeeded'
+        403:
+          description: 'Analysis does not exist or user is not authorized to view analysis'
+
+  /{projectId}/{z}/{x}/{y}/:
+    x-resource: Mosaic
+    get:
+      produces:
+        - image/png
+      summary: Visualize a mosaic of a project
+      tags:
+        - Mosaic
+      parameters:
+        - $ref: '#/parameters/zoom'
+        - $ref: '#/parameters/x'
+        - $ref: '#/parameters/y'
+        - $ref: '#/parameters/token'
+        - $ref: '#/parameters/mapToken'
+        - $ref: '#/parameters/redBandOverride'
+        - $ref: '#/parameters/greenBandOverride'
+        - $ref: '#/parameters/blueBandOverride'
+        - $ref: '#/parameters/projectId'
+      responses:
+        200:
+          description: OK
+        401:
+          description: 'Authentication failed -- no authentication methods succeeded'
+        403:
+          description: 'Project does not exist or user is not authorized to view analysis'
+
+  /{projectId}/histogram/:
+    x-resource: Mosaic
+    get:
+      summary: 'Obtain the project mosaic histogram'
+      tags:
+        - Mosaic
+      parameters:
+        - $ref: '#/parameters/token'
+        - $ref: '#/parameters/mapToken'
+        - $ref: '#/parameters/redBandOverride'
+        - $ref: '#/parameters/greenBandOverride'
+        - $ref: '#/parameters/blueBandOverride'
+        - $ref: '#/parameters/projectId'
+      responses:
+        200:
+          description: OK
```

Comment:
Might be good to actually include the format for the histogram response

Reply:
Will do :ok_hand:

Reply:
Actually now that I'm thinking about trying to do it, it's kind of hard. Because it's an `int -> int` map but not every int will necessarily be present in the response, it'd end up something like:

```yaml
0:
  type: int
  required: false
1:
  type: int
  required: false
...
```

Reply:
Ok did it -- thanks, python `yaml` lib